### PR TITLE
fix: update prettier

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

**Comment by czosel** [comment]
Friendly ping @cseufert :slightly_smiling_face: 

cc @alexander-akait @loilo

File: src/comments.js [hunk 1]
```diff
@@ -8,15 +8,8 @@ const {
...
-const {
-  concat,
-  join,
-  indent,
-  hardline,
-  cursor,
-  lineSuffix,
-  breakParent,
-} = require("prettier").doc.builders;
+const { concat, join, indent, hardline, cursor, lineSuffix, breakParent } =
```
Comment (alexander-akait) [inline src/comments.js:11]: As I remeber concat is deprecated, but it is not a problem for such update

**Review by alexander-akait — APPROVED** [review]
Also it will be good for dependeices, because now you can have only one prettier in your node_moduels

File: src/comments.js (hunk 1 above)
Comment (czosel) [reply src/comments.js:11]: That's right - we'll tackle that in the context of #1922 in a follow-up PR :+1:

**Comment by czosel** [comment]
Thanks for the review @alexander-akait! :+1:

**Pull request merged**